src/views/HomeRedesign/CDKTypeTabs.tsx

```diff
@@ -59,9 +60,21 @@ const PackageTabPanel = forwardRef<PackageTabProps & TabPanelProps, "div">(
 
 export const CDKTypeTabs: FunctionComponent = () => {
   const anyCDKType = useCatalogResults({ limit: 4 });
-  const awscdk = useCatalogResults({ cdkType: CDKType.awscdk, limit: 4 });
-  const cdk8s = useCatalogResults({ cdkType: CDKType.cdk8s, limit: 4 });
-  const cdktf = useCatalogResults({ cdkType: CDKType.cdktf, limit: 4 });
+  const awscdk = useCatalogResults({
```

Comment:
One more nitpick is that we should also have the button that says `See All {CDKType} Constructs` link to the search results with the same Download sorting

Reply:
Added.